brainframe/cli/commands/update.py

```diff
@@ -25,7 +25,19 @@ def update():
     existing_version = version.parse(existing_version_str)
     requested_version = version.parse(requested_version_str)
 
-    if not args.force:
+    force_downgrade = False
+    if args.noninteractive:
+        # Use the --force flag to decide if downgrades are allowed
+        force_downgrade = args.force
+    else:
```

Comment:
Why would the `--force` flag not work when interactive?

Reply:
Because the CLI goes into non-interactive mode if any flags are provided.